package.json

```diff
@@ -56,7 +57,7 @@
     "morgan": "^1.9.0",
     "passport-local-mongoose": "^4.4.0",
     "pify": "^3.0.0",
-    "xlsx": "^0.12.7",
+    "xlsjs": "^1.0.1",
```

Comment:
On npm it says the package is deprecated and we should use xlsx
https://www.npmjs.com/package/xlsjs